websites/A/AsphaltEvents/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Senka Wolf",
+      "id": "170854083840704512"
+    },
+    "contributors": {
+      "name": "DubStepMad",
+      "id": "118810952530395136"
+    },
+    "url": [
+      "asphalt.events",
+      "asphaltevents.com"
+    ],
+    "description": {
+      "en": "Modern and feature-packed convoy/event advertising service for TruckersMP."
+    },
+    "service": "Asphalt Events",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/MXhf8fA.png",
+    "thumbnail": "https://i.imgur.com/MXhf8fA.png",
+    "color": "#3490DC",
```

Comment:
thumbnail should be a promotional banner of the website, or a screenshot of the website, not the logo.

Reply:
Ah thank you, will resolve that now.

Reply:
Resolved in updated pull request.